config/initializers/assets.rb

```diff
@@ -3,6 +3,8 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
+Rails.application.config.assets.precompile += %w( print.css )
```

Comment:
[This line](https://github.com/PacemakerConf/site/blob/72e3d1f9a7864ca0149a841de85427f9feeb1920/config/initializers/assets.rb#L13) should be uncommented and modified. Do not duplicate code.